docs/build/core-contracts/10-nft-storefront.md

```diff
@@ -74,15 +74,15 @@ The `NFTStorefrontV2` contract doesn’t support selling an NFT for multiple dif
 
 ![scenario_1](./scenario_1.png)
 
-Putting an NFT on sell called listing, seller can create a listing using [sell_item](https://github.com/onflow/nft-storefront/blob/main/transactions/sell_item.cdc) transaction by providing some required details to list an NFT, i.e. Receiving currency type, [Capability](https://cadence-lang.org/docs/language/capabilities) from where NFT will be deducted etc. If interested look [here](<#fun-createListing()>) for more details.
+Putting an NFT on sell called listing, seller can create a listing using [sell_item](https://github.com/onflow/nft-storefront/blob/main/transactions/sell_item.cdc) transaction by providing some required details to list an NFT, i.e. Receiving currency type, [Capability](https://cadence-lang.org/docs/language/capabilities) from where NFT will be deducted etc. If interested look [here](<#fun-createlisting>) for more details.
```

Comment:
Not sure if now is the right time, but I'd change these reference-style links to make it easier to read and edit the raw files.

That will also help require us to use more descriptive text for the links, which is better for ADA than [here], and I _think_ impacts SEO.  Though I'm far from an expert on the latter.

Reply:
converted links to reference style and fixed some links